zoraxy/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: zoraxy_server_1
+      APP_PORT: 8000
+      PROXY_AUTH_ADD: "false"
+  server:
+    image: zoraxydocker/zoraxy:v3.1.2@sha256:38f23c5a507542bbd6eac0492bf6456be2004015594ca270a1a5998c8a377b74
+    restart: on-failure
+   #ports: -> The proxy service is listening on port 8400
+      # 8480:80
+      # 8443:443
+      # 8400:8000

```

Comment:
Since Zoraxy is a reverse proxy doesn't it need to expose http and https ports for its basic functionality?

User's are going to need to open ports on their router and then port forward to Zoraxy running on their Umbrel. You can bind to free ports that haven't been taken by other apps yet. So something like:

```yml
ports:
  - 41080:80
  - 41443:443
```

Then right at the top of the app description you could note which ports are for which protocol.